MAINTAINERS

```diff
@@ -124,8 +124,93 @@ the relevant operators.
 
 * If the change affects the governance, philosophy, goals or principles of the project,
 it must be approved by BDFL.
+
+* A pull request can be in 1 of 5 distinct states, for each of which there is a corresponding label
+that needs to be applied. `Rules.review.states` contains the list of states with possible targets
+for each.
 """
 
+		# Triage
+		[Rules.review.states.0-triage]
```

Comment:
btw- big +1 to starting each label with the step number - makes it much clearer to follow and understand the flow.